addon/session-stores/cookie.js

```diff
@@ -38,12 +38,16 @@ const persistingProperty = function(beforeSet = function() {}) {
   ```js
   // app/controllers/login.js
   export default Ember.Controller.extend({
-    rememberMe: false,
-
-    _rememberMeChanged: Ember.observer('rememberMe', function() {
-      const expirationTime = this.get('rememberMe') ? (14 * 24 * 60 * 60) : null;
-      this.set('session.store.cookieExpirationTime', expirationTime);
-    }
+    rememberMe: computed({
```

Comment:
Just had a look at the  [Guides](https://guides.emberjs.com/v3.0.0/object-model/computed-properties/#toc_setting-computed-properties), too and this example seems good 👍